guides/getting-started/set-up/cheqd/index.md

```diff
@@ -0,0 +1,76 @@
+import DocCardList from '@theme/DocCardList';
+
+# Cheqd
+
+[cheqd](https://github.com/cheqd/sdk) cheqd is a blockchain network, built in the Cosmos ecosystem for self-sovereign identity. It bridges the DeFi ecosystem with the decentralised identity ecosystem, and provides an implementation of the the new Ledger [Agnostic AnonCreds Specification (v1.0)](https://hyperledger.github.io/anoncreds-spec/).
+
+
+### Installing cheqd
+
+When using Aries Framework JavaScript with the cheqd, there's a few extra dependencies that need to be installed. We need to install the `@aries-framework/cheqd` package, which implements the needed interfaces for the agent.
+
+To enable react-native support we need to follow the steps below
+
+<DocCardList />
+
+# React Native
+
+In the package.json file add the below code snippet, which replaces the cosmjs dependencies witht he cosmjs-rn packages
+```json
+ "resolutions": {
+    "@cosmjs/amino": "npm:@cosmjs-rn/amino@^0.27.1",
+    "@cosmjs/encoding": "npm:@cosmjs-rn/encoding@^0.27.1",
+    "@cosmjs/math": "npm:@cosmjs-rn/math@^0.27.1",
+    "@cosmjs/stargate": "npm:@cosmjs-rn/stargate@^0.27.1",
+    "@cosmjs/tendermint-rpc": "npm:@cosmjs-rn/tendermint-rpc@^0.27.1",
+    "@cosmjs/utils": "npm:@cosmjs-rn/utils@^0.27.1",
+    "@cosmjs/proto-signing": "npm:@cosmjs-rn/proto-signing@^0.27.1",
+    "@cosmjs/crypto": "npm:@cosmjs-rn/crypto@^0.27.1"
+  }
```

Comment:
Does this also work with npm overrides? If you look at the other guides at the `ref-napi` one, we have tabs for setting it up for NPM and Yarn

Reply:
This is complete